src/main/java/com/google/swarm/tokenization/common/InspectData.java

```diff
@@ -41,13 +39,22 @@
 @SuppressWarnings("serial")
 public class InspectData
     extends DoFn<KV<String, Iterable<Table.Row>>, KV<String, InspectContentResponse>> {
+
   public static final Logger LOG = LoggerFactory.getLogger(InspectData.class);
   private final String projectId;
   private final String inspectTemplateName;
   private final PCollectionView<List<String>> headerColumns;
   private transient DlpServiceClient dlpServiceClient;
   private transient InspectContentRequest.Builder requestBuilder;
 
+  private final Counter numberOfRowsInspected =
+      Metrics.counter(InspectData.class, "numberOfRowsInspected");
+
+  private final Counter numberOfDlpApiCalls =
+      Metrics.counter(InspectData.class, "numberOfDlpApiCalls");
+
+  private final Counter numberOfBadRows = Metrics.counter(InspectData.class, "NumberOfBadRows");
```

Comment:
same thing here as well. Also please note this will be overwritten with next release when I include beam transform. @BhupiSindhwani

Reply:
@santhh - I pushed the new commit with the comments in place for the counters